pennylane/queuing.py

```diff
@@ -220,6 +221,9 @@ def _append(self, obj, **kwargs):
     def _remove(self, obj):
         self.queue.remove(obj)
 
+    append = _append
```

Comment:
```suggestion
    # Overwrite the inherited class methods, so that if queue.append is called,
    # it is appended to the instantiated queue (rather than being added to the
    # currently active queuing context, which may be a different queue).
    append = _append
```